CesiumGltfReader/src/applyKhrTextureTransform.cpp

```diff
@@ -118,5 +118,7 @@ void applyKhrTextureTransform(Model& model) {
       }
     }
   }
+
```

Comment:
Remove extra space?

Reply:
I think it's ok to keep the extra space, it's consistent with the the files below

Reply:
Sorry, I had split view enabled on Github so it looked like there were two line breaks in a row. 😬 This is fine!